v5.1.1.sql

```diff
@@ -323,3 +323,15 @@ ALTER TABLE openchpl.certified_product DROP COLUMN IF EXISTS terms_of_use_url;
 ALTER TABLE openchpl.pending_certified_product DROP COLUMN IF EXISTS terms_of_use_url;ALTER TABLE openchpl.test_tool add column retired boolean NOT NULL DEFAULT false;
 UPDATE openchpl.test_tool set retired = true where name = 'Transport Testing Tool';
 UPDATE openchpl.test_tool set retired = true where name = 'Transport Test Tool';
+
```

Comment:
Nothing should be added to v5.1.1.sql. This file is the file that updates a database from v5.1.0 to v5.1.1, and should never be changed after it's committed and used during a deployment. Instead, this insert code should be in the `dev/openchpl_preload.sql` file around line 1180, where the other ROLEs are added during a "standup" of the database